.github/workflows/ssgts-cs8.yaml

```diff
@@ -0,0 +1,178 @@
+      - name: Test if there are no content changes
+        run: echo "::set-output name=CTF_OUTPUT_SIZE::$(stat --printf="%s" output.json)"
+        id: ctf
+      - uses: actions/upload-artifact@v2
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        with:
+          name: output.json
+          path: output.json
+      - name: Print changes to content detected if any
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        run: cat output.json
+      - name: Get product attribute
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        id: product
+        uses: notiz-dev/github-action-json-property@release
+        with:
+          path: 'output.json'
+          prop_path: 'product'
+      - name: Build product
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        run: ./build_product ${{steps.product.outputs.prop}} --datastream-only
+      - uses: actions/upload-artifact@v2
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        with:
+          name: ssg-${{steps.product.outputs.prop}}-ds.xml
+          path: build/ssg-${{steps.product.outputs.prop}}-ds.xml
+  validate-ubuntu:
+    name: Run Tests
+    needs: build-content
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install Deps
+        uses: mstksg/get-package@master
+        with:
+          apt-get: cmake ninja-build libopenscap8 libxml2-utils expat xsltproc python3-jinja2 python3-yaml ansible-lint podman
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get cached CTF output
+        uses: actions/download-artifact@v2
+        id: get_ctf_output
+        with:
+          name: output.json
+        # continue even if the file is unavailable that
+        # means there are no changes detected by CTF in the previous job
+        continue-on-error: true
+      - name: Test if there are no content changes
+        if: ${{ steps.get_ctf_output.outcome == 'success' }}
+        run: echo "::set-output name=CTF_OUTPUT_SIZE::$(stat --printf="%s" output.json)"
+        id: ctf
+      - name: Print changes to content detected if any
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        run: cat output.json
+      - name: Generate id_rsa key
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        run: ssh-keygen -N '' -t rsa -f ~/.ssh/id_rsa
+      - name: Build test suite container
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        run: podman build --build-arg "CLIENT_PUBLIC_KEY=$(cat ~/.ssh/id_rsa.pub)" -t ssg_test_suite -f test_suite-cs8
+        working-directory: ./Dockerfiles
+      - name: Get oscap-ssh
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        run: |
+          wget https://raw.githubusercontent.com/OpenSCAP/openscap/maint-1.2/utils/oscap-ssh
```

Comment:
We should rather use `maint-1.3` branch here.

Reply:
I'm going to change and see how it goes.